packages/ui-stencil/src/components/internal/orama-search-results/orama-search-results.tsx

```diff
@@ -27,9 +27,24 @@ export class SearchResults {
   @Prop() setChatTerm: (term: string) => void
   @Prop() loading = false
   @Prop() error = false
-  @Prop() highlight?: HighlightOptions | false = false
+  @Prop() highlightTitle?: HighlightOptions | false = false
+  @Prop() highlightDescription?: HighlightOptions | false = false
 
-  private highlighter?: Highlight
+  private highlighterTitle?: Highlight
+  private highlighterDescription?: Highlight
+
+  private getSourceUrl = (path: string) => {
+    if (!path) {
+      return '#'
+    }
+
+    if (this.sourceBaseUrl) {
+      const sourceUrl = new URL(path, this.sourceBaseUrl)
+      return sourceUrl.toString()
+    }
+
+    return path
+  }
```

Comment:
This should also solve this isse https://github.com/orgs/oramasearch/discussions/831